### ISSUE TITLE
Elden Ring DLSS FG mod +dlssg-to-fsr3 ?

### ISSUE THREAD

**Issue by trxsudo** [Nukem9/dlssg-to-fsr3#270]
https://github.com/Nukem9/dlssg-to-fsr3/wiki/Elden-Ring
as it states here the "DLSS FG mod", which DLSS FG mod is it exactly? and any instructions on how to install all of this? thanks

**Comment by ImASadBoy** [comment]
> https://github.com/Nukem9/dlssg-to-fsr3/wiki/Elden-Ring as it states here the "DLSS FG mod", which DLSS FG mod is it exactly? and any instructions on how to install all of this? thanks

PureDraks's DLSS mod for Elden Ring, to install that follow PureDark instructions.
https://www.patreon.com/posts/elden-ring-has-0-86786103

**Comment by ImASadBoy** [comment]
But I should add that it really is not worth it, there's too much flicker on the UI

**Comment by TimmyP777** [comment]
Totally worth it, 2x fps increase, slight UI blur, played for hours yesterday

**Comment by ImASadBoy** [comment]
Slight UI blur? I'm getting UI all over the place xD

**Comment by darthmalak1986** [comment]
> Totally worth it, 2x fps increase, slight UI blur, played for hours yesterday

i couldnt make it work keeps crashing

Can you gave me instructions over how do you make it work?

Thanks!

**Comment by ImASadBoy** [comment]
> > Totally worth it, 2x fps increase, slight UI blur, played for hours yesterday
> 
> i couldnt make it work keeps crashing
> 
> Can you gave me instructions over how do you make it work?
> 
> Thanks!

Follow PureDrak's mod instructions and then move Nukem9's files  next to the .exe

**Comment by TimmyP777** [comment]
> Slight UI blur? I'm getting UI all over the place xD

Yeah its not that bad. I was already at 60 and now its 110ish. Its noticeable when your looking, but played for hours the other day and I didnt even notice it.

Its part of the mod. I think the expansion is gonna add official DLSS3 support, so itll be fine then.

**Comment by ImASadBoy** [comment]
> > Slight UI blur? I'm getting UI all over the place xD
> 
> Yeah its not that bad. I was already at 60 and now its 110ish. Its noticeable when your looking, but played for hours the other day and I didnt even notice it.
> 
> Its part of the mod. I think the expansion is gonna add official DLSS3 support, so itll be fine then.

Thinking FromSoftware is going to innovate their tech? Think again xD

**Comment by macangel** [comment]
The DLSS, does it have options like it does in games that support it natively? Like can you choose Quality, Balance, and Performance, or are you stuck with a single version? Same with the frame generation with the DLSS, can you turn that part on and off, just use the DLSS, or just use the frame generation?

**Comment by ImASadBoy** [comment]
> The DLSS, does it have options like it does in games that support it natively? Like can you choose Quality, Balance, and Performance, or are you stuck with a single version? Same with the frame generation with the DLSS, can you turn that part on and off, just use the DLSS, or just use the frame generation?

It has even more options such as Native DLAA and the ability to choose between presets for DLSS
